src/com/facebook/buck/event/listener/AbstractConsoleEventBusListener.java

```diff
@@ -197,7 +197,7 @@ public void setPublicAnnouncements(ImmutableList<String> announcements) {
   protected String formatElapsedTime(long elapsedTimeMs) {
     long minutes = elapsedTimeMs / 60_000L;
     String seconds = TIME_FORMATTER.format(locale, elapsedTimeMs / 1000.0 - (minutes * 60));
-    return String.format(minutes == 0 ? "%s" : "%2$dm%1$s", seconds, minutes);
+    return String.format(minutes == 0 ? "%s" : "%2$dm %1$s", seconds, minutes);
```

Comment:
Looks reasonable - fwiw, though, I went with this format because it's pretty much what the `time` command uses. I'm not against this change, but I do think there is some value to using what people are used to seeing.